internal/monitors/cadvisor/converter/converter.go

```diff
@@ -49,16 +57,24 @@ type machineInfoMetric struct {
 	getValues func(s *info.MachineInfo) metricValues
 }
 
+type podStatusMetric struct {
+	name      string
+	valueType datapoint.MetricType
+	getValues func(s *stats.FsStats) metricValues
+}
+
 // CadvisorCollector metric collector and converter
 type CadvisorCollector struct {
-	infoProvider            InfoProvider
-	containerMetrics        []containerMetric
-	containerSpecMetrics    []containerSpecMetric
-	containerSpecMemMetrics []containerSpecMetric
-	containerSpecCPUMetrics []containerSpecMetric
-	machineInfoMetrics      []machineInfoMetric
-	sendDP                  func(*datapoint.Datapoint)
-	defaultDimensions       map[string]string
+	infoProvider               InfoProvider
+	containerMetrics           []containerMetric
+	containerSpecMetrics       []containerSpecMetric
+	containerSpecMemMetrics    []containerSpecMetric
+	containerSpecCPUMetrics    []containerSpecMetric
+	machineInfoMetrics         []machineInfoMetric
+	podEphemeralStorageMetrics []podStatusMetric
+	sendDP                     func(*datapoint.Datapoint)
+	defaultDimensions          map[string]string
+	groupsEnablerHelper        GroupsEnablerHelper
```

Comment:
This can be the bool based on `HasEnabledMetricInGroup` output.

Reply:
Updated to pass in just the boolean for this group as we discussed